Sadly, the warnings only show up in the raw logs of travis: 

```
warning: [options] source value 1.6 is obsolete and will be removed in a future release
warning: [options] target value 1.6 is obsolete and will be removed in a future release
warning: [options] To suppress warnings about obsolete options, use -Xlint:-options.
/home/travis/build/java-json-tools/msg-simple/src/main/java/com/github/fge/msgsimple/provider/LoadingMessageSourceProvider.java:272: warning: [FutureReturnValueIgnored] Return value of methods returning Future must be checked. Ignoring returned Futures suppresses exceptions thrown from the code that completes the Future.
        scheduled.scheduleAtFixedRate(runnable, duration, duration, unit);
                                     ^
    (see https://errorprone.info/bugpattern/FutureReturnValueIgnored)
  Did you mean to remove this line?
/home/travis/build/java-json-tools/msg-simple/src/main/java/com/github/fge/msgsimple/bundle/MessageBundle.java:210: warning: [JdkObsolete] StringBuffer performs synchronization that is usually unnecessary; prefer StringBuilder.
                .format(params, new StringBuffer(pattern.length()), null)
                                ^
    (see https://errorprone.info/bugpattern/JdkObsolete)
5 warnings
```

Why don't the compilation warnings show up in the "install" phase?

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,4 +2,7 @@ language: java
 jdk:
   - openjdk8
   - openjdk11
+install: {}
+script:
+  - ./gradlew assemble check
 

